maquetado/index.html

```diff
@@ -7,40 +7,86 @@
         <img class="logo" src="./assets/react_icon.svg" alt="React">
         <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
+        <h4 class="technology-title bold">Release:2013</h4>
         <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg" alt="React">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
-        <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+        <img class="logo" src="./assets/angular_icon.svg" alt="angular">
+        <h4 class="technology-title">AngularJS</h4>
+        <h4 class="technology-title bold">Release:2010</h4>
+        <p class="technology-description">
+          AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.
+        </p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg" alt="React">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
-        <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+        <img class="logo" src="./assets/vue_icon.svg" alt="Vue">
+        <h4 class="technology-title">VueJS</h4>
+        <h4 class="technology-title bold">Release:2014</h4>
+        <p class="technology-description">
+          Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.
+        </p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg" alt="React">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
-        <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+        <img class="logo" src="./assets/elm_icon.svg" alt="Elm">
+        <h4 class="technology-title">Elm</h4>
+        <h4 class="technology-title bold">Release:2016</h4>
+        <p class="technology-description">
+          Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.
+        </p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg" alt="React">
-        <h4 class="technology-title">ReactJS</h4>
-        <h4 class="technology-release">Release:2013</h4>
-        <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+        <img class="logo" src="./assets/polymer_icon.svg" alt="Polymer">
```

Comment:
Same as the `tecnologia` PR. I would change this class name into something more specific since you could have more than on logo on your screen. Maybe `tech-logo`